doc/ug/introduction.tex

```diff
@@ -321,6 +321,19 @@ \section{Requirements}
   version 1.2).
 \end{description}
 
+\subsection{Installing Requirements on Ubuntu 16.04 LTS}
+\label{sec:installing_requirements}
+\index{installing_requirements}
+
+To make ESPResSo run on Ubuntu 16.04 LTS its dependencies can be installed with:
```

Comment:
This is not specific to Ubuntu or that version. I guess that it should also work on all supported Ubuntu versions (12.04, 14.04, 15.10 and 16.06) and Debian versions (7 and 8). Please check the package names on these distributions and make sure they already have the `apt` command (older versions may require `apt-get`). Then make the text more generic as appropriate.